### PR TITLE
Runtime error trace format

### LINKED ISSUE #109: Error message trace format
```
Runtime Error: not callable: int
    at mod1.tengo:3:4
    at mod2.tengo:19:7
    at main.tengo:4:1
```

### PR DESCRIPTION
**Pull request by earncef**
Fixed #109 and cleaned up the code a little.

### CODE REVIEW

File: runtime/vm.go [hunk 1]
```diff
@@ -335,14 +337,14 @@ mainloop:
 
 			res, e := (*left).BinaryOp(token.AndNot, *right)
 			if e != nil {
-				filePos := v.fileSet.Position(v.curFrame.fn.SourceMap[v.ip])
+				filePos = v.fileSet.Position(v.curFrame.fn.SourceMap[v.ip])
 				if e == objects.ErrInvalidOperator {
 					err = fmt.Errorf("%s: invalid operation: %s &^ %s",
```
Comment (d5) [inline runtime/vm.go]: remove `%s: ` and `filePos` from here?

**Comment by d5** [comment]
This will fix #109.

**Comment by earncef** [comment]
I’ll do these shortly.

**Comment by earncef** [comment]
Done

**Comment by d5** [comment]
This is awesome! I will merge it. Thanks!

**Pull request merged**